src/minidump.rs

```diff
@@ -866,35 +866,35 @@ Memory
 }
 
 /// An iterator over `MinidumpMemory`s.
-pub struct MemoryRegions<'b, 'a>
+pub struct MemoryRegions<'iter, 'data>
 where
-    'a: 'b,
+    'data: 'iter,
 {
-    iter: Box<dyn Iterator<Item = &'b MinidumpMemory<'a>> + 'b>,
+    iter: Box<dyn Iterator<Item = &'iter MinidumpMemory<'data>> + 'iter>,
 }
 
-impl<'b, 'a> Iterator for MemoryRegions<'b, 'a>
+impl<'iter, 'data> Iterator for MemoryRegions<'iter, 'data>
 where
-    'a: 'b,
+    'data: 'iter,
 {
-    type Item = &'b MinidumpMemory<'a>;
+    type Item = &'iter MinidumpMemory<'data>;
 
-    fn next(&mut self) -> Option<&'b MinidumpMemory<'a>> {
+    fn next(&mut self) -> Option<&'iter MinidumpMemory<'data>> {
         self.iter.next()
     }
 }
 
-impl<'a> MinidumpMemoryList<'a> {
+impl<'mdmp> MinidumpMemoryList<'mdmp> {
```

Comment:
If this naming is adopted perhaps this should be done for all of the stream types.  On the other hand perhaps the doc comment on `.get_stream()` is sufficient and we should revert this rename?

Reply:
This seems very sensible and I would be happy with that. With a single named lifetime things are usually fine but as you've demonstrated here, adding a second one without naming them clearly leaves room for confusion!